src/utils/__tests__/component-utils.spec.ts

```diff
@@ -160,4 +161,39 @@ describe('component-utils', () => {
       message: 'done',
     });
   });
+
+  it('should return status.lastPromotedImage when lastPromotedImage is available', () => {
+    const mockComponent = {
+      status: {
+        lastPromotedImage: 'test-url',
+      },
+    } as unknown as ComponentKind;
+
+    expect(renderHook(() => getLastestImage(mockComponent)).result.current).toEqual('test-url');
```

Comment:
use renderHook only for testing react-hooks, not for functions.

```suggestion
    expect(getLastestImage(mockComponent)).toEqual('test-url');
```